package.json

```diff
@@ -5,7 +5,7 @@
 	"scripts": {
 		"build": "tsc && node build.js",
 		"lint": "prettier --write .",
-		"test": "jest"
+		"test": "tsc && jest"
```

Comment:
It looks like all test are TypeScript files. Consider using the `ts-jest` preset for `jest` to avoid the extra `tsc` step.

See [the documentation](https://jestjs.io/docs/getting-started#via-ts-jest).